.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         id: test
         uses: gradle/gradle-build-action@3fbe033aaae657f011f88f29be9e65ed26bd29ef # v2.3.3
         with:
-          arguments: testReleaseUnitTest
+          arguments: testReleaseUnitTest -x :screenshots-test:testReleaseUnitTest
```

Comment:
This is error prone. If we don't want to run "release" tests of this module, we should simply disable this buildType entirely.

Reply:
Done in a792de44